src/workloads/execution/ShardedLookup.yml

```diff
@@ -0,0 +1,116 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+
+  Database: &Database test
+
+  LocalCollection: &LocalCollection Collection0
+  LocalNamespace: &LocalNamespace test.Collection0
+
+  ForeignCollection: &ForeignCollection Collection1
+  ForeignNamespace: &ForeignNamespace test.Collection1
+
+  DocumentCount: &DocumentCount 5000
+  BatchSize: &BatchSize 1000
+  NumChunks: &NumChunks 5
```

Comment:
Do you need all of these to be defined as global defaults? I'm not sure what the preferred YAML style is here, but as a general rule I try not to use global when it's not necessary to do so.

Another way to define these would be right when you use it for the first time. For example, instead of defining Database here, you could have on line 30: `enableSharding: &db test` and then reuse it with `*db` in other phases, even within other actors.

Reply:
The global default style is not required, but I went with it because I thought it was more clear, particularly for the local/foreign collection/namespace values, to define up-front. But, I see we don't do this elsewhere in the query workload code, so I've removed them in the newest commit.